src/status_im/navigation/roots.cljs

```diff
@@ -49,3 +49,19 @@
    {:root {:stack {:children [{:component {:name    :syncing-results
                                            :id      :syncing-results
                                            :options (options/dark-root-options)}}]}}}})
+
+(defn old-roots
```

Comment:
could you elaborate on why we still need that?

Reply:
Hi both, biometric login and logout were using an intermediate root screen and removal of that caused #19236 bug.

I can still remove them, means event also which opens this intermediate root. But not sure we want that, as this shows a progress screen.
https://github.com/status-im/status-mobile/blob/develop/src/legacy/status_im/ui/screens/progress/views.cljs

Reply:
I mean we should decide if we will continue to use it, it should be in the new roots, etc, overwise remove it

Reply:
https://github.com/status-im/status-mobile/assets/17097240/37cf71e4-ad80-4314-987b-bee13fe84f4a

This is how it looks.

Reply:
> I mean we should decide if we will continue to use it, it should be in the new roots, etc, overwise remove it

Lets keep this screen as temporary until we have something from design team. Should I still move this to new roots, even if its temporary and using legacy screen/comopnents?

Reply:
ok let's merge this to fix the bug, and then decide